cluster-provision/images/vm-image-builder/fedora-with-test-tooling/cloud-config

```diff
@@ -36,7 +36,10 @@ write_files:
           WantedBy=cloud-init.service
 runcmd:
   - sudo systemctl daemon-reload
-  - sudo dnf install -y kernel-modules-$(uname -r) qemu-guest-agent stress-ng dmidecode virt-what vm-dump-metrics
+  - sudo dnf install -y http://ccrma.stanford.edu/planetccrma/mirror/fedora/linux/planetccrma/16/i386/planetccrma-repo-1.1-3.fc16.ccrma.noarch.rpm
```

Comment:
Shouldn't you use `http://ccrma.stanford.edu/planetccrma/mirror/fedora/linux/planetccrma/32/x86_64/planetccrma-repo-1.1-3.fc32.ccrma.noarch.rpm` which is built for Fedora 32?

The rpm you're referencing is built for Fedora 16.

Reply:
What is this package? why is it needed?

Reply:
@acardace Thanks! I will update.
@ormergi  This package installs the repo that contains the kernel-rt and other real time packages. I want to use the existing fedora-with-testing-tools for the real time functional tests. If this change has no impact to non-real time functional tests that leverage on this image, then we will avoid having to create a new container disk image just for real time functional tests.